src/components/calcite-slider/calcite-slider.stories.ts

```diff
@@ -48,6 +48,22 @@ export const Range = (): string => html`
   ></calcite-slider>
 `;
 
+export const RangeLabeledTicks = (): string => html`
```

Comment:
I think we need more stories for Screener to catch the rest of the cases where labels may overlap. I had shared in a comment a link to a previous demo page with each case.

Reply:
I think this story should catch the expected behavior. This issue focus on tick labels irrespective of the thumb handles  positions. One other example i can think of is the mirrored range slider.

Reply:
If you look at [the following](https://codepen.io/jcfranco/pen/mdMoKJv?editors=1000), you can see the difference between the additional stories I suggested and the ones in this PR: 

![Screen Shot 2021-11-19 at 6 42 32 PM](https://user-images.githubusercontent.com/197440/142711818-ad8a8d5f-0442-4f87-9c45-f4cd31cdc1cb.png)

With the additional stories, we have more Screener coverage on visual changes to labels and thumbs at the edges. If I remember correctly, the entries from the demo page covered a different code path of the label positioning. I don't believe this has changed, so that is why I was suggesting adding these.

Reply:
Make sense. The current demo's follow a different path.